protobuf.lua

```diff
@@ -72,7 +72,7 @@ project "protobuf"
   -- configurations
   -- -------------------------------------------------------------
 
-  if (os.is("windows") and not _TARGET_IS_WINRT and not _TARGET_IS_WINPHONE) then
+  if (os.is("windows") and not _TARGET_IS_WINUWP) then
```

Comment:
looks like this commit does a lot more than it says it does.

Reply:
The commit does mention some lua normalization cleanup was done.  Would you prefer this in a separate commit?

Reply:
As this is a public repo, I'd leave it up to you to decide on what will be best.